programs/crowdfund/src/lib.rs

```diff
@@ -122,7 +122,7 @@ pub mod crowdfund {
 
         let vault_sol_after = ctx.accounts.pda_vault.lamports();
         let vault_token_after = ctx.accounts.pda_vault_ata.amount;
-        let pda_address = ctx.accounts.pda_vault.key;
+        //let pda_address = ctx.accounts.pda_vault.key;
```

Comment:
any reason to leave this in?

Reply:
Really just for debugging because it feeds into the below commented out printline